flexibleengine/data_source_flexibleengine_vpc_subnet_v1_test.go

```diff
@@ -87,6 +87,7 @@ data "flexibleengine_vpc_subnet_v1" "by_id" {
 }
 
 data "flexibleengine_vpc_subnet_v1" "by_name" {
+	id = "${flexibleengine_vpc_subnet_v1.subnet_1.id}"
```

Comment:
why adding id here?

Reply:
@niuzhenguo removed id and change it for name only.